lib/testing/src/lib/protractor/core/pages/settings.page.ts

```diff
@@ -104,6 +107,8 @@ export class SettingsPage {
         await this.setLogoutUrl(logoutUrl);
         await this.clickApply();
         await browser.sleep(1000);
+        await this.clickSignInSSO();
```

Comment:
This is for sure wrong - Log in without CodeFlow means we are logging in without SSO probably , or the implicit Flow is always active here? what about the basic login ? what about the silent login enabled?